plugins/base/src/main/kotlin/translators/documentables/briefFromContentNodes.kt

```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.dokka.base.translators.documentables
+
+import org.jetbrains.dokka.pages.ContentGroup
+import org.jetbrains.dokka.pages.ContentNode
+import org.jetbrains.dokka.pages.ContentText
+import org.jetbrains.dokka.pages.TextStyle
+
+fun briefFromContentNodes(description: List<ContentNode>): List<ContentNode> {
+    val firstSentenceRegex = Regex("^((?:[^.?!]|[.!?](?!\\s))*[.!?])")
```

Comment:
```suggestion
    val firstSentenceRegex = """^((?:[^.?!]|[.!?](?!\s))*[.!?])""".toRegex()
```

Reply:
I know it is only moved from another place, but this is not the idiomatic way of doing that.